### ISSUE TITLE
Missing server-side setup.

### ISSUE THREAD

**Issue by s-mv** [web-are-bears/rnd-fixer#1]
The server-side code is missing for RnD Fixer. It needs to be added to the monorepo since [rnd-server](https://github.com/web-are-bears/rnd-server) is obsolete now.

TODOs:
- Add `server` directory to the repository with the server side code.
- Move CONTRIBUTING.md outside of `client` and update it according to the requirements.
